lib/rubocop/cop/rspec/expect_actual.rb

```diff
@@ -41,11 +41,33 @@ class ExpectActual < Cop
           regexp
         ].freeze
 
-        def_node_matcher :expect_literal, '(send _ :expect $#literal?)'
+        SUPPORTED_MATCHERS = %i[eq eql equal be].freeze
+
+        def_node_matcher :expect_literal, <<~PATTERN
+          (send
+            (send nil? :expect $#literal?)
+            #{Runners::ALL.node_pattern_union}
+            {
+              (send (send nil? $:be) :== $_)
+              (send nil? $:be $...)
+              (send nil? $_ $_ ...)
+            }
+          )
+        PATTERN
 
         def on_send(node)
           expect_literal(node) do |argument|
-            add_offense(argument)
+            add_offense(node, location: argument.source_range)
+          end
+        end
+
+        def autocorrect(node)
+          actual, matcher, expected = expect_literal(node)
+          lambda do |corrector|
+            return unless SUPPORTED_MATCHERS.include?(matcher)
+            return if expected.is_a?(Array) # `be` without args
```

Comment:
The comment is not precise, as arguments will also result in having an array. The `:==` case does not result in an array because of the `$_` (while `$...` results in an array). So you are not correcting `be`, unless followed by `==`

Reply:
Nice catch, the `be` with an argument case completely slipped my mind 👍 
I assumed I'm using `$...` as a placeholder just to balance the number of captures in patter union clause, and it will always be `[]` for `be`.